bundle/config/mutator/python/python_mutator.go

```diff
@@ -258,6 +258,12 @@ func (m *pythonMutator) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagno
 		return newRoot, nil
 	})
 
+	// don't instrument deprecated phases
+	if m.phase == PythonMutatorPhaseLoadResources || m.phase == PythonMutatorPhaseApplyMutators {
```

Comment:
Can you include a comment explaining why we sum across phases?

Reply:
Added below